recipes/qt/6.x.x/conanfile.py

```diff
@@ -472,6 +472,8 @@ def build(self):
             build_env["MAKEFLAGS"] = "j%d" % tools.cpu_count()
             build_env["PKG_CONFIG_PATH"] = [self.build_folder]
             if self.settings.os == "Windows":
+                if not "PATH" in build_env:
+                    build_env["PATH"] = []
```

Comment:
it fixes the following error on mingw, and when cross compiling from linux towindows:
```
ERROR: qt/6.0.2: Error in build() method, line 475
	build_env["PATH"].append(os.path.join(self.source_folder, "qt6", "gnuwin32", "bin"))
	KeyError: 'PATH'
```
it happens because in these configurations, `build_env` is initialized as an empty map on line 471